src/IconButton/README.md

```diff
@@ -28,16 +31,20 @@ notes: ''
 ```jsx live
 () => {
   const icon = FontAwesome.faTimes;
+  const variants = ["brand", "primary", "secondary", "success", "warning", "danger", "light", "dark"];
 
   return (
     <div className="d-flex">
-      <IconButton icon={icon} alt='Close' onClick={() => {}} variant="primary" />
-      <IconButton icon={icon} alt='Close' onClick={() => {}} variant="secondary" />
-      <IconButton icon={icon} alt='Close' onClick={() => {}} variant="success" />
-      <IconButton icon={icon} alt='Close' onClick={() => {}} variant="warning" />
-      <IconButton icon={icon} alt='Close' onClick={() => {}} variant="danger" />
-      <IconButton icon={icon} alt='Close' onClick={() => {}} variant="light" />
-      <IconButton icon={icon} alt='Close' onClick={() => {}} variant="dark" />
+      {variants.map((variant) => (
```

Comment:
For the docs, I prefer to have them written out as they were; I find it easier to use when I'm developing.

Reply:
Good point!